app/src/main/java/io/github/trojan_gfw/igniter/MainActivity.java

```diff
@@ -355,8 +353,11 @@ public void onRun() {
     protected void onResume() {
         super.onResume();
 
-        // use handler to getPrimaryClip
-        mHandler.post(new Runnable() {
+        detectClipboard();
```

Comment:
Its naming is too vague, how about 'CheckTrojanURLFromClipboard'?

Reply:
😂It's better